cmd/kedge/term.go

```diff
@@ -49,6 +49,10 @@ func waitForAny(ctx context.Context, signals ...os.Signal) {
 		}
 
 		logrus.Infof("Graceful Shutdown after %s timed out, re-sending SIGTERM.", gotSignal.String())
-		syscall.Kill(syscall.Getpid(), syscall.SIGTERM)
+		p, err := os.FindProcess(os.Getpid())
+		if err != nil {
+			logrus.Errorf("Can't find process %v", err)
+		}
+		p.Signal(syscall.SIGTERM)
```

Comment:
should we use `Kill` instead of sending sigterm - there was some weird behaviour on windows with sigterms, right?

Reply:
It seems like SIGTERM issue was affecting `cmd`, I think this `os.Process.Signal` might trigger appropriate os dependent things under the hood. Kind of how `file.Chmod` does not work on Windows but `os.Chmod` does.

Since the tests pass I will merge this for now.